### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by dlecan**
Hi,

- EN / DE improvements
- New FR translation
- Explain how to use Unikname Connect OIDC provider (mostly used in France)

Thank you

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -77,4 +77,13 @@ server {
...
 }
+
+- [Unikname Connect](https://unikname.com)
+
+  - Name: `Connect with your private @unikname`
+  - Authorize URL: `https://connect.unikname.com/oidc/authorize`
+  - Token URL: `https://connect.unikname.com/oidc/accessToken`
+  - Userinfo URL: `https://connect.unikname.com/oidc/profile`
+  - Userinfo ID: `sub`
+  - OAuth Scopes: `openid email`
```
Comment (dominik-th) [inline docs/faq.md:88]: This part is written in the nginx code block. Can you move it between Gitlab and Microsoft Azure AD?

File: lang/de.json [hunk 2]
```diff
@@ -13,15 +13,15 @@
         "SettingUserinfoUrl": "Userinfo URL",
         "SettingUserinfoUrlHelp": "z.B. https://<USERNAME>.eu.auth0.com/userinfo",
         "SettingUserinfoId": "Userinfo ID",
-        "SettingUserinfoIdHelp": "Name des Feldes, in dem die Benutzer-ID enthalten ist. Normalerweise, für OpenID Connect Dienste wie Auth0, ist das 'sub'. Github gibt die eindeutige Benutzer-ID in dem Feld 'id' an.",
+        "SettingUserinfoIdHelp": "Name des Feldes, in dem die Benutzer-ID enthalten ist. Normalerweise, für OpenID Connect Dienste wie Auth0, Keycloak oder Unikname Connnect, ist das 'sub'. Github gibt die eindeutige Benutzer-ID in dem Feld 'id' an.",
```
Comment (dominik-th) [inline lang/de.json:16]: The help text should be kept short.
Auth0 is the most popular commercial auth service and Keycloak the most popular selfhosted, so they should be enough.

File: lang/en.json [hunk 3]
```diff
@@ -15,15 +15,15 @@
         "SettingUserinfoUrl": "Userinfo URL",
         "SettingUserinfoUrlHelp": "e.g. https://<USERNAME>.eu.auth0.com/userinfo",
         "SettingUserinfoId": "Userinfo ID",
-        "SettingUserinfoIdHelp": "Name of the unique user id field in the userinfo response. Usually for OpenID Connect services like Auth0 this is 'sub'. Github provides the user id in 'id'.",
+        "SettingUserinfoIdHelp": "Name of the unique user id field in the userinfo response. Usually for OpenID Connect services like Auth0, Keycloak or Unikname Connect this is 'sub'. Github provides the user id in 'id'.",
```
Comment (dominik-th) [inline lang/en.json:18]: Same as in the German translation

File: docs/faq.md (hunk 1 above)
Comment (dlecan) [reply docs/faq.md]: Moved between Gitlab and Azure

File: lang/de.json (hunk 2 above)
Comment (dlecan) [reply lang/de.json]: Removed

File: lang/en.json (hunk 3 above)
Comment (dlecan) [reply lang/en.json]: Removed, in French too

**Comment by dlecan** [comment]
Changes done!
Thank you for reviewing this PR

**Comment by dlecan** [comment]
@dominik-th anything else to modify?
thank you

**Comment by dominik-th** [comment]
No its fine. Thanks for your contribution

**Pull request merged**